app/controllers/courses_controller.rb

```diff
@@ -20,12 +20,11 @@ def decorated_courses
   end
 
   def ordered_courses
-    Course.order(:position).includes(:lessons, sections: [:lessons])
+    Course.order(:position)
   end
 
+  # TODO: remove me and just use current_user everywhere
   def set_user
-    if user_signed_in?
-      @user = User.includes(:lesson_completions).find(current_user.id)
-    end
+    @user = current_user
```

Comment:
I left this for now backwards compatibility, since a lot of existing code relies on it. However, as I mentioned in the PR, I suggest that someone else take a pass to remove this from the code base, now that it is (I believe) no longer necessary.